clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -2331,15 +2331,15 @@
         "body": [
             "ROW START ${1:TBL|TBFnam}."
         ],
-        "description": "[DaCo] ROW START",
+        "description": "[DaCo] Set the line-currency (RCU) to zero in order to be able to process the  \n table sequentially from the beginning.\n After this command is given, the ROW NEXT command can be used to move the\n currency 1 line forward.\n",
```

Comment:
Set the line-currency (RCU) to zero in order to process the \n table sequentially from the beginning.\n After this command is given, the ROW NEXT command can be used to move the\n currency 1 line forward.\n

Reply:
@zeibura This is the description I got from my team. No worries, I will confirm with the team. However this is an example description, so that team can visualize how description will displayed for statement.

Reply:
I will update the description as per your suggestion.

Reply:
Done